src/providers/bitcoin/BitcoinJsLibSwapProvider.js

```diff
@@ -102,42 +102,7 @@ export default class BitcoinJsLibSwapProvider extends Provider {
   }
 
   async refundSwap (initiationTxHash, recipientAddress, refundAddress, secretHash, expiration) {
-    return this._redeemSwap(initiationTxHash, recipientAddress, refundAddress, secretHash, expiration, false)
-  }
-
-  async _redeemSwap (initiationTxHash, recipientAddress, refundAddress, secretParam, expiration, isClaim) {
```

Comment:
there were some references to old code here. Since the code is just copied, thought i'd remove it